howso/ablation.amlg

```diff
@@ -111,23 +111,21 @@
 
 	;reduce the trained data by removing cases which have an influence weight entropy that falls above
 	; a threshold.
-	;
-	;parameters:
-	; distribute_weight_feature: optional, default !autoAblationWeightFeature. name of feature whose values to use as case weights
-	; features: default all trained features. a list of features to use when computing influence weight entropies
-	; influence_weight_entropy_threshold: optional, default !autoAblationInfluenceWeightEntropyThreshold. cases with
-	; 	influence weight entropy above this quantile will be removed
-	; skip_auto_analyze: default false. whether to skip auto analyzing as cases are removed
 	#reduce_data
 	(declare
 		(assoc
 			;{type "string"}
+			;default !autoAblationWeightFeature. name of feature whose values to use as case weights
 			distribute_weight_feature !autoAblationWeightFeature
 			;{type "list" values "string"}
+			;default all trained features. a list of features to use when computing influence weight entropies
 			features !trainedFeatures
 			;{type "number"}
+			;default !autoAblationInfluenceWeightEntropyThreshold. cases with
```

Comment:
remove 'cases with' at the end, it's a duped on the next line

Reply:
Removed in